Application/Application.iml

```diff
@@ -7,5 +7,6 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" name="com.google.code.gson:gson:2.8.2" level="project" />
```

Comment:
Why is this added here? isn't this part of the parsing?

Reply:
it is because Pontus imported the library to his branch yesterday, let's use this iml file in develop and delete the rest